05/app.js

```diff
@@ -1,79 +1,87 @@
 const stats = {
     paragraphs: {
-        'p1': 0,
+        p1: 0,
     },
     links: {
-        '/dolor.html': 0,
-    }
+        "/dolor.html": 0,
+    },
 };
-console.log(stats.links["/dolor.html"]);
 
 /* tutaj umieść swój kod */
-const paragraph = document.querySelectorAll('p');
-const a = document.getElementsByClassName('link');
-const arr = Array.from(a)
-console.log(arr);
+const paragraph = document.querySelectorAll("p");
+const a = document.getElementsByClassName("link");
+const arr = Array.from(a);
 paragraph.forEach(function(element) {
-    element.addEventListener('click', function() {
-        stats.paragraphs['p1']++;
-        console.log(stats.paragraphs);
-    }, true);
-
+    element.addEventListener(
+        "click",
+        function(e) {
+            const tagName = e.target.tagName;
+            if (tagName.includes("P")) {
+                stats.paragraphs["p1"]++;
```

Comment:
Dobry kierunek, ale zauważ, że każdy paragraf ma zapisany w atrybucie `data-id` inną wartość.
To ona powinna być wstawiona zamiast zawsze "p1". Pamiętaj, że nie możesz zrobić `++` na właściwości, która nie istnieje.